Towny/src/main/java/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -1769,7 +1771,9 @@ public static boolean isTownSpawnPaidToTown() {
 
 	public static double getNationNeutralityCost(Nation nation) {
 		double cost = nation.getNationLevel().peacefulCostMultiplier() * getNationNeutralityCost();
-		return isNationNeutralityCostMultipliedByNationTownAmount() ? cost * nation.getTowns().size() : cost;
+		NationNeutralityCostCalculationEvent event = new NationNeutralityCostCalculationEvent(nation, cost);
```

Comment:
The cost here should be `isNationNeutralityCostMultipliedByNationTownAmount() ? cost * nation.getTowns().size() : cost` and not done after the event is fired.

The event should be fired with the true cost that will be charged.